### ISSUE TITLE
new entry 

### ISSUE THREAD

**Issue by tremblap** [flucoma/learn-website#201]
a single entry in learn, maybe in articles, where we talk about where is the long list of made-with, so if someone searches for made-with they are pointed at the right place. that should go in the 3rd tab (stories, projects, whatever we call it)

**Comment by jdchart** [comment]
Right, just pointing to the www or also with a list (i believe we wanted to avoid duplicates)?

**Comment by jamesb93** [comment]
Can you clarify what this suggestion is? I'm finding it a little hard to understand @tremblap
